snugget_load.py

```diff
@@ -147,12 +140,23 @@ def getFilterFieldName(shapefile):
 
 def getShapefileFilter(shapefile, filterVal):
   fieldName = getFilterFieldName(shapefile)
-  kwargs = {fieldName: filterVal}
-  if shapefile.objects.filter(**kwargs).exists():
-    return shapefile.objects.get(**kwargs)
+  if fieldName == 'rast':
+    for tile in shapefile.objects.defer("rast").all():
+      if int(filterVal) in tile.rast.bands[0].data():
+        return int(filterVal)
+    print(
```

Comment:
wait a second, it looks like this message always prints. Also, you're not just checking the first tile, right?

Reply:
It won't print if a value was found, because as soon as this function finds the value in any tile it returns it.  But now that you mention this, I can use that loop to also keep track of the model-wide min and max values, like in `findAllFilterVals()`

Reply:
It will now report the min & max for the whole raster, not just the first tile.